custom_components/tuya_local/devices/alecoair_d12_home_dehumidifier.yaml

```diff
@@ -0,0 +1,235 @@
+            value: 0
+            icon: mdi:cancel
+            icon_priority: 3
+          - dps_val: 1 hour
+            value: 1
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 2 hours
+            value: 2
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 3 hours
+            value: 3
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 4 hours
+            value: 4
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 5 hours
+            value: 5
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 6 hours
+            value: 6
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 7 hours
+            value: 7
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 8 hours
+            value: 8
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 9 hours
+            value: 9
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 10 hours
+            value: 10
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 11 hours
+            value: 11
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 12 hours
+            value: 12
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 13 hours
+            value: 13
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 14 hours
+            value: 14
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 15 hours
+            value: 15
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 16 hours
+            value: 16
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 17 hours
+            value: 17
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 18 hours
+            value: 18
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 19 hours
+            value: 19
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 20 hours
+            value: 20
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 21 hours
+            value: 21
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 22 hours
+            value: 22
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 23 hours
+            value: 23
+            icon: "mdi:store-clock"
+            icon_priority: 3
+          - dps_val: 24 hours
+            value: 24
+            icon: "mdi:store-clock"
+            icon_priority: 3
+  - entity: select
```

Comment:
This should be a fan entity.  Make this the speed (mapped to values 50 and 100), and make dp 1 the switch.  Using standard entities where appropriate like this will help with things like voice control.